.travis.yml

```diff
@@ -44,17 +29,9 @@ install:
   - git clone https://github.com/google/re2.git
   - make -C re2
   - env DESTDIR="$HOME" make -C re2 install
-  - git clone https://github.com/google/googletest.git
-before_script:
-  - git submodule update
-  - GTEST_DIR=${TRAVIS_CI_PATH}/googletest/googletest
-  - g++ -std=c++11 -isystem ${GTEST_DIR}/include -I${GTEST_DIR} -pthread -c ${GTEST_DIR}/src/gtest-all.cc
-  - ar -rv libgtest.a gtest-all.o
-  - cd ${GTEST_DIR}/make
-  - make
 script:
-  - cd ${TRAVIS_CI_PATH}/cpp
-  - env CXXFLAGS="-I$HOME/usr/local/include -I${GTEST_DIR}/include -I${GTEST_DIR} -I${GTEST_DIR}/make" LDFLAGS="-L$HOME/usr/local/lib -L$(pwd) -L${GTEST_DIR}/include -L${GTEST_DIR} -L${GTEST_DIR}/make" CPPFLAGS="-I${GTEST_DIR}/include -I${GTEST_DIR} -I${GTEST_DIR}/make" GYP_GENERATORS="ninja"  gyp --depth .
+  - cd cpp
+  - env CXXFLAGS="-I$HOME/usr/local/include" LDFLAGS="-L$HOME/usr/local/lib" GYP_GENERATORS="ninja" gyp --depth .
   - ninja -C out/Default
```

Comment:
My investigation result: the only way to make the gtests work is to stop using ninja, bc it uses the older versions. As far as I searched/tested they don't work together.

Reply:
@rsolomakhin WDYT? Also @parastoog do you know why we made the change to SUITE from CASE?

Reply:
It's explained in this document:
https://docs.google.com/document/d/1bcccx4SlrBwZmlMXNhp_v8WUdl_ph3CSvM99VsPzTpc/edit